packages/arb-token-bridge-ui/src/components/TransferPanel/AdvancedSettings.tsx

```diff
@@ -154,6 +161,17 @@ export const AdvancedSettings = ({
           {!error && warning && (
             <span className="text-xs text-yellow-500">{warning}</span>
           )}
+          {destinationAddress && !error && (
+            <ExternalLink
+              className="arb-hover mt-2 flex w-fit items-center"
+              href={`${getExplorerUrl(
+                (isDepositMode ? l2 : l1).network.id
+              )}/address/${destinationAddress}`}
+            >
+              <ArrowDownTrayIcon height={16} className="mr-2 -rotate-90" />
```

Comment:
The font size seems a little bigger than expected. IMHO the link text should be of size `text-sm or xs` just like the Learn more text under LINK div.

<img width="844" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/7558499/bd54f59f-91ff-45f8-bfe6-522a149a96d3">

Reply:
Discussed with the design and updated